vagrant/provision-liquid.sh

```diff
@@ -31,6 +31,7 @@ fi
 
 cp examples/liquid.ini .
 
+./liquid deploystats
```

Comment:
So this prints the total resource needs of the deployment, not any real stats. Let's call it `resources` or something.